--- sitaramc-gitolite-9b5793f/contrib/adc/get-rights-and-owner.in-perl.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/contrib/adc/get-rights-and-owner.in-perl	2011-02-14 11:59:29.565791964 +0100
@@ -4,6 +4,7 @@
 use warnings;
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # get the repo name
--- sitaramc-gitolite-9b5793f/contrib/adc/gl-reflog.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/contrib/adc/gl-reflog	2011-02-14 11:59:29.565791964 +0100
@@ -41,6 +41,7 @@
 $limit ||= 10;
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 my ($perm, $creator, $wild) = &repo_rights($repo);
--- sitaramc-gitolite-9b5793f/hooks/common/update.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/hooks/common/update	2011-02-14 11:59:29.569125380 +0100
@@ -36,6 +36,7 @@
 die "parse $ENV{GL_RC} failed: "       . ($! or $@) unless do $ENV{GL_RC};
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 my ($perm, $creator, $wild) = &repo_rights($ENV{GL_REPO});
--- sitaramc-gitolite-9b5793f/src/gl-auth-command.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/src/gl-auth-command	2011-02-14 11:59:29.572458795 +0100
@@ -45,6 +45,7 @@
 $bindir =~ s/\/[^\/]+$//;
 $bindir = "$ENV{PWD}/$bindir" unless $bindir =~ /^\//;
 unshift @INC, $bindir;
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # ask where the rc file is, get it, and "do" it
--- sitaramc-gitolite-9b5793f/src/gl-compile-conf.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/src/gl-compile-conf	2011-02-14 11:59:29.575792210 +0100
@@ -61,6 +61,7 @@
 $bindir =~ s/\/[^\/]+$//;
 $bindir = "$ENV{PWD}/$bindir" unless $bindir =~ /^\//;
 unshift @INC, $bindir;
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # ask where the rc file is, get it, and "do" it
--- sitaramc-gitolite-9b5793f/src/gl-install.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/src/gl-install	2011-02-14 11:59:29.575792210 +0100
@@ -29,6 +29,7 @@
 my $bindir = $0;
 $bindir =~ s/\/[^\/]+$//;
 unshift @INC, $bindir;
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # ask where the rc file is, get it, and "do" it
--- sitaramc-gitolite-9b5793f/src/gl-setup-authkeys.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/src/gl-setup-authkeys	2011-02-14 11:59:29.579125625 +0100
@@ -31,6 +31,7 @@
 $bindir =~ s/\/[^\/]+$//;
 $bindir = "$ENV{PWD}/$bindir" unless $bindir =~ /^\//;
 unshift @INC, $bindir;
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # prevent newbie from running it accidentally and clobbering his authkeys
--- sitaramc-gitolite-9b5793f/src/gl-time.old	2011-01-15 14:32:34.000000000 +0100
+++ sitaramc-gitolite-9b5793f/src/gl-time	2011-02-14 11:59:29.582459040 +0100
@@ -23,6 +23,7 @@
 $bindir =~ s/\/[^\/]+$//;
 $bindir = "$ENV{PWD}/$bindir" unless $bindir =~ /^\//;
 unshift @INC, $bindir;
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 
 # ---------------------------------------------------------------
