--- sitaramc-gitolite-80d28e6/contrib/adc/get-rights-and-owner.in-perl.old	2011-03-05 12:59:20.000000000 +0100
+++ sitaramc-gitolite-80d28e6/contrib/adc/get-rights-and-owner.in-perl	2011-04-15 19:53:08.182357528 +0200
@@ -7,6 +7,7 @@
 die "ENV GL_BINDIR not set\n" unless $ENV{GL_BINDIR};
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 gitolite->import;
 
--- sitaramc-gitolite-80d28e6/contrib/adc/git.old	2011-03-05 12:59:20.000000000 +0100
+++ sitaramc-gitolite-80d28e6/contrib/adc/git	2011-04-15 19:53:08.189022618 +0200
@@ -85,6 +85,7 @@
     die "ENV GL_BINDIR not set\n" unless $ENV{GL_BINDIR};
 
     unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
     require gitolite or die "parse gitolite.pm failed\n";
     gitolite->import;
 
--- sitaramc-gitolite-80d28e6/contrib/adc/gl-reflog.old	2011-03-05 12:59:20.000000000 +0100
+++ sitaramc-gitolite-80d28e6/contrib/adc/gl-reflog	2011-04-15 19:53:08.192355164 +0200
@@ -44,6 +44,7 @@
 $limit ||= 10;
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 gitolite->import;
 
--- sitaramc-gitolite-80d28e6/contrib/adc/hub.old	2011-03-05 12:59:20.000000000 +0100
+++ sitaramc-gitolite-80d28e6/contrib/adc/hub	2011-04-15 19:53:08.195687710 +0200
@@ -64,6 +64,7 @@
 usage() unless ($dispatch{$cmd});
 
 unshift @INC, $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite or die "parse gitolite.pm failed\n";
 gitolite->import;
 
--- sitaramc-gitolite-80d28e6/src/gl-query-rc.old	2011-03-05 12:59:20.000000000 +0100
+++ sitaramc-gitolite-80d28e6/src/gl-query-rc	2011-04-15 19:53:08.195687710 +0200
@@ -14,6 +14,7 @@
     $0 =~ m|^(/)?(.*)/| and $ENV{GL_BINDIR} = ($1 || "$ENV{PWD}/") . $2;
 }
 use lib $ENV{GL_BINDIR};
+use lib "/usr/share/gitolite/lib";
 require gitolite_rc;
 gitolite_rc->import;
 
